### PR TITLE
Fix stray closing paren in hex serialization

### PR DESCRIPTION
**Pull request by raphlinus**
I was printing debug colors and noticed an extraneous parenthesis. Also adds test.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ This release has an [MSRV][] of 1.82.
 ### Changed
 
 * The `mul_alpha` method was renamed to `multiply_alpha`. ([#65][] by [@waywardmonkeys][])
-* Stray parenthesis in hex serialization of Rgba8 fixed (NUMBER_TODO by [@raphlinus][])
+* Stray parenthesis in hex serialization of Rgba8 fixed (#[#78][] by [@raphlinus][])
```
Comment (waywardmonkeys) [inline CHANGELOG.md:32]: Stray # here.

**Review by waywardmonkeys — APPROVED** [review]
LGTM once the stray # is fixed!

**Comment by waywardmonkeys** [comment]
Oh, you'll also need to add the link anchor for @raphlinus alphabetically in the existing list of people.

**Review by waywardmonkeys — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ This release has an [MSRV][] of 1.82.
 ### Changed
 
 * The `mul_alpha` method was renamed to `multiply_alpha`. ([#65][] by [@waywardmonkeys][])
+* Stray parenthesis in hex serialization of Rgba8 fixed ([#78][] by [@raphlinus][])
```
Comment (DJMcNab) [inline CHANGELOG.md:32]: For future reference, I'd probably put this in the `### Fixed` section

Comment (raphlinus) [reply CHANGELOG.md:32]: Yeah, feel free to roll that into a future PR. I'm not sure why I didn't do that. Also I noticed after that the convention is a final period, which I missed.

**Pull request merged**